CMakeLists.txt

```diff
@@ -14,6 +14,10 @@ read_version("${CMAKE_CURRENT_SOURCE_DIR}/version.pri" VER_MAJOR VER_MINOR VER_B
 
 project(qBittorrent VERSION ${VER_MAJOR}.${VER_MINOR}.${VER_BUGFIX}.${VER_BUILD})
 
+# check for invalid compiler version/CXX standard as early as possible
+include(MacroQbtCXXCompilerAndModeCheck)
```

Comment:
The very last comment: a function is inside a file named "Macro".

Reply:
@zeule fixed.